.github/CONTRIBUTING.md

```diff
@@ -27,6 +26,12 @@ If you're not used to this workflow with git, you can start with some [docs from
 
 ## Tests
 
+You have the option to test your changes locally by running the pipeline. For receiving warnings about process selectors and other `debug` information, it is recommended to use the debug profile. Execute all the tests with the following command:
+
+```bash
+nf-test test --profile debug,test,docker --verbose
```

Comment:
is this command working?

Reply:
Don't think so. I just tried it, and got:

```
🚀 nf-test 0.8.4
https://code.askimed.com/nf-test
(c) 2021 - 2024 Lukas Forer and Sebastian Schoenherr

Warning: This pipeline has no nf-test config file.
Error: No tests or test directories containing scripts that end with *.test provided
```

I don't know where that is stuff in `CONTRIBUTING.md` is coming from. Can I just delete it?